npm/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.4.7
```

Comment:
Don't think this should be here? This is a scala property, but is in the `/npm/project` directory. There is a `build.properties` in the `/project` directory. You can probably update the sbt version in that file, as it is currently set to: `sbt.version=1.3.8`

Reply:
Ah, this was a mistake; it must have gotten accidentally git added